pkg/cmd/step/create/step_create_meta_pipeline.go

```diff
@@ -61,14 +57,10 @@ var (
 type StepCreatePipelineOptions struct {
 	*opts.CommonOptions
 
-	SourceURL    string
-	Branch       string
-	PullRefs     string
-	PipelineKind string
```

Comment:
I'm not sure we can get rid of `PipelineKind`, `Context`, etc as options - but that may just be me thinking about things from the perspective of the `step_create_task` tests.

Reply:
`Context` is still there, that can indeed not be derived. PipelineKind I derive from the pull ref atm, similar to what the pipeline runner does as well. If need be, this option could be brought back. It seems to me though that right now we only have release or pullrequest. Having two options you can also create inconsitent runs, by for example specifying a pull ref for a pull request, but use kind 'release'. That does not make much sense to me.

Reply:
Ah, I misread and just saw the line deleting `Context`, not the new line.

Maybe we should force `pullRequest` as the `Kind` if `PullRefs` is not empty, even if you specify a `Kind`? That would allow `feature` to still work. Or we can just ignore it for now and deal with it later if it comes up. =)